### ISSUE TITLE
Translate exercise filter headers

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/flutter#208]
At the moment, the filter headers used for the exercise filters are hard coded in English (in the provider). These strings should be translated like everything else in the application

**Comment by rolandgeider** [comment]
![image](https://user-images.githubusercontent.com/2280729/181476008-4aaebb1a-7bdb-405b-b9da-a93cd2b9be55.png)

**Comment by rolandgeider** [comment]
Hi! Sure, go ahead.

This is still in the `feature/exercise-crowdsourcing`, you can use the test server for the backend:

https://wger.rge.uber.space
username: user
password: flutteruser

In the workout tab, go to the menu on the top right to exercise list. The provider is the "add exercise provider" and instead of muscle groups we could use "category" which is used in other places and  should already be translated

**Comment by sandimilohanic** [comment]
Is there maybe a user interface for adding new translations?

**Comment by sandimilohanic** [comment]
do you mean this group of muscles to set the category?

![Screenshot_66](https://user-images.githubusercontent.com/7765448/195848677-e172114f-e040-4a81-8a09-1d5e75e8e73c.png)

**Comment by rolandgeider** [comment]
No, the ones when you click on that funnel icon next to the text field. Also, this was recently moved to master, no need to work in the branch anymore.

**Comment by rolandgeider** [comment]
> @rolandgeider Is there maybe a user interface for adding new translations?

We have setup synchronisation with weblate https://hosted.weblate.org/engage/wger/ Is this what you mean?

**Comment by sandimilohanic** [comment]
> We have setup synchronisation with weblate https://hosted.weblate.org/engage/wger/ Is this what you mean?

yes i mean for that.

**Comment by sandimilohanic** [comment]
Regarding for this issue, i need to add two translations in the backend app into the file "i18n.tpl". Can I add them?

{% translate "Category" %}
{% translate "Equipment" %}
